bushpig.gemspec

```diff
@@ -44,6 +44,6 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'connection_pool', '~> 2.2'
   spec.add_dependency 'json', '>= 1.8'
-  spec.add_dependency 'redis', '~> 3.3'
+  spec.add_dependency 'redis', '~> 4.1.4'
```

Comment:
I would relax this to just `>= 4.1`

gems, as a rule, should not have restrictive dependencies. This makes them very difficult to integrate into apps where versions are more tightly controlled (as you just found out with toolkit and this gem)